src/App.jsx

```diff
@@ -1,15 +1,24 @@
 import React from 'react';
 import './App.scss';
+import Sum from './components/Sum';
 
-const App = () => (
+const App = (a, b) => (
```

Comment:
```suggestion
const App = () => (
```
Why do you need these arguments here? You don't use them anywhere